### DISCUSSION TITLE
官方提供的股票查询例子中，function call为什么 THUDM/chatglm3-6b 支持 THUDM/chatglm3-6b-128k 模型不支持呢.

### DISCUSSION THREAD

**Discussion by goqw** [zai-org/ChatGLM3#1127] [Q&A] [answered]
https://github.com/THUDM/ChatGLM3/tree/main/tools_using_demo
使用此Tool Demo，调用不同的模型，发现
THUDM/chatglm3-6b 支持
THUDM/chatglm3-6b-128k 模型不支持
请问是怎么回事，是否有支持的>8K Token的模型呢。

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
只有 6b支持哦，32k 128k都不支持
